### PR TITLE
Change `--tag` option from bool to template in project:bump command

### LINKED ISSUE #197: Add tag prefix to settings for `project bump`
Reference: https://github.com/c4urself/bump2version#tag_name-

### PR DESCRIPTION
**Pull request by b0g3r**
Closes #197 

I have questions about three things:
1. CONTRIBUTE.md doesn't say anything about running modified dephell for manual testing, only running checking and testing. Now I'm using `poetry run dephell` command, but I'm not sure about it.
2. I don't see any tests for argument schema validation. Why?
2. I have no idea about git tag creation testing :( I tried to search, but found only tests for git package resolving.

### CODE REVIEW

**Comment by orsinium** [comment]
Beautiful. Thank you!

I think, we can use `{version}` instead of `{new_version}`. Also, if placeholder is not specified, we can interpret it as a prefix. For example, `--tag="v."`

**Comment by b0g3r** [comment]
Good proposal, I implemented the prefix feature! But I concerned about testing git tag feature, how can I test it?

**Comment by orsinium** [comment]
Oh, that's a long story. We have to make temporary directory, `git init`, run command and check commits. However, we have no this kind of tests yet, it's in future plans. I've commented it in #167, and then we lost contributor. So, I can accept PR without tests and add it in the scope of future works about this kind of tests.

**Comment by b0g3r** [comment]
Added tests!

File: docs/cmd-project-bump.md [hunk 1]
```diff
@@ -23,10 +23,11 @@ Command steps:
 1. Write new version in source code. DepHell looks for `__version__` variable in project source and writes new version in it.
 1. Write new version in `from` file.
 
-Also, the command adds git tag if `--tag` option (or `tag = True` in the config) is specified:
+Also, the command adds git tag if `--tag` option (or `tag = <your_template>` in the config) is specified as template.
+Template can be string with `{version}` placeholder (e.g. `v.{version}`) or just prefix string (e.g. `v.`)
 
 ```bash
-$ dephell project bump --tag minor
```
Comment (orsinium) [inline docs/cmd-project-bump.md]: You've missed rule here, it's required. Give it back, please.

File: dephell/commands/project_bump.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def get_parser(cls) -> ArgumentParser:
         builders.build_output(parser)
         builders.build_api(parser)
         builders.build_other(parser)
-        parser.add_argument('--tag', action='store_true', help='create git tag')
+        parser.add_argument('--tag', action='store', help='create git tag')
```
Comment (orsinium) [inline dephell/commands/project_bump.py]: `action` is `store` by default, isn't it?

File: dephell/commands/project_bump.py [hunk 3]
```diff
@@ -138,10 +139,19 @@ def _update_metadata(self, root, loader, new_version) -> bool:
...
+            # for example: `--tag=v.{version}` will be transform to `v.3.0.1`
+            tag_name = tag_prefix_or_template.format(version=new_version)
+        else:
+            # if we don't found placeholder, we use template as a prefix
+            # for example: `--tag=v.` will be transform to `v.3.0.1`
+            tag_name = '{prefix}{version}'.format(prefix=tag_prefix_or_template, version=new_version)
         project = Path(self.config['project'])
         if not (project / '.git').exists():
-            return
+            self.logger.error("project doesn\'t contains .git in root folder, cannot create git tag")
```
Comment (orsinium) [inline dephell/commands/project_bump.py]: "project doesn\'t contain .git in the root folder, cannot create git tag"

File: dephell/commands/project_bump.py [hunk 4]
```diff
@@ -138,10 +139,19 @@ def _update_metadata(self, root, loader, new_version) -> bool:
             stream.write(new_content)
         return True
 
-    def _add_git_tag(self, paths, new_version):
+    def _add_git_tag(self, paths, new_version, tag_prefix_or_template: str) -> bool:
+        if '{version}' in tag_prefix_or_template:
```
Comment (orsinium) [inline dephell/commands/project_bump.py]:
```python
if '{version}' not in template:
    template += '{version}'
tag_name = template.format(version=new_version)
```

**Comment by orsinium** [comment]
Perfect, thank you :) Tonight Travis CI has troubles with Linux build, so, I'll merge it tomorrow, if tests works

**Comment by orsinium** [comment]
Thank you, nice job.

If you'll have time and willing to help more, we have a bug #127 related to the usage of `.git` in some tests. If you propagate the way of testing from this PR on other git-based commands, it will work. Feel free to contact me or open an issue if you have some questions or bugs.

**Pull request merged**